.trivyignore

```diff
@@ -0,0 +1,9 @@
+# List of vulnerabilities to ignore for the trivy scan
```

Comment:
i'm wondering if we can get rid of this file too since it looks to be the same as on template-infra

Reply:
As we continue to build in this repo, there will probably be other things we need to recast or safelist that are just for this repo, so I think we should keep the config files here since they'd fix those issues